src/category/category.controller.ts

```
@@ -2,8 +2,9 @@ import { Controller, Get, Post, Body, Patch, Param, Delete, UseGuards, Request,
 import { CategoryService } from './category.service';
 import { CreateCategoryDto } from './dto/create.category.dto';
 import { UpdateCategoryDto } from './dto/update.category.dto';
-import { ApiBearerAuth, ApiBody, ApiOperation, ApiProperty } from '@nestjs/swagger';
+import { ApiBearerAuth, ApiBody, ApiOperation, ApiProperty, ApiResponse } from '@nestjs/swagger';
```

ESLint

[error] Replace `·ApiBearerAuth,·ApiBody,·ApiOperation,·ApiProperty,·ApiResponse·` with `⏎··ApiBearerAuth,⏎··ApiBody,⏎··ApiOperation,⏎··ApiProperty,⏎··ApiResponse,⏎`

(prettier/prettier)